internal/pkg/aws/ecs/ecs.go

```diff
@@ -333,13 +333,47 @@ func (e *ECS) ActiveClusters(arns ...string) ([]string, error) {
 
 	var active []string
 	for _, cluster := range resp.Clusters {
-		if aws.StringValue(cluster.Status) == clusterStatusActive {
+		if aws.StringValue(cluster.Status) == statusActive {
 			active = append(active, aws.StringValue(cluster.ClusterArn))
 		}
 	}
 
 	return active, nil
+}
 
+// ActiveServices returns the subset of service arns that have an ACTIVE status.
+// Note that all services should be in the same cluster.
+func (e *ECS) ActiveServices(serviceARNs ...string) ([]string, error) {
+	var prevSvcArn *ServiceArn
+	for _, arn := range serviceARNs {
+		svcArn, err := ParseServiceArn(arn)
+		if err != nil {
+			return nil, err
+		}
+		if prevSvcArn != nil && prevSvcArn.clusterName != svcArn.clusterName {
+			return nil, fmt.Errorf("service %q and service %q should be in the same cluster", prevSvcArn.String(), svcArn.String())
```

Comment:
Hi! I keep getting this error when doing `copilot service show`, and it's referencing old (deleted) clusters and services. Is it possible that this cluster check should be done after inactive services are filtered out?